packages/esm-generic-patient-widgets-app/src/obs-graph/obs-graph.component.tsx

```diff
@@ -71,38 +71,45 @@ const ObsGraph: React.FC<ObsGraphProps> = ({ patientUuid }) => {
   };
 
   return (
-    <div className={styles.graphContainer}>
-      <div className={styles.conceptPickerTabs}>
-        <label className={styles.conceptLabel} htmlFor="concept-tab-group">
-          {t('displaying', 'Displaying')}
-        </label>
-        <Tabs id="concept-tab-group" className={styles.verticalTabs} type="default">
-          <TabList className={styles.tablist} aria-label="Obs tabs">
-            {config.data.map(({ concept, label }, index) => {
-              return (
-                <Tab
-                  key={index}
-                  className={`${styles.tab} ${styles.bodyLong01} ${
-                    selectedConcept.label === label && styles.selectedTab
-                  }`}
-                  onClick={() =>
-                    setSelectedConcept({
-                      label,
-                      uuid: concept,
-                    })
-                  }
-                >
-                  {label}
-                </Tab>
-              );
-            })}
-          </TabList>
-        </Tabs>
+    <>
+      <div className={styles.graphContainer}>
+        <div className={styles.conceptPickerTabs}>
+          <label className={styles.conceptLabel} htmlFor="concept-tab-group">
+            {t('displaying', 'Displaying')}
+          </label>
+          <Tabs id="concept-tab-group" className={styles.verticalTabs} type="default">
+            <TabList className={styles.tablist} aria-label="Obs tabs">
+              {config.data.map(({ concept, label }, index) => {
+                return (
+                  <Tab
+                    key={index}
+                    className={`${styles.tab} ${styles.bodyLong01} ${
+                      selectedConcept.label === label && styles.selectedTab
+                    }`}
+                    onClick={() =>
+                      setSelectedConcept({
+                        label,
+                        uuid: concept,
+                      })
+                    }
+                  >
+                    {label}
+                  </Tab>
+                );
+              })}
+            </TabList>
+          </Tabs>
+        </div>
+        <div className={styles.lineChartContainer}>
+          <LineChart data={chartData.flat()} options={chartOptions} />
+        </div>
       </div>
-      <div className={styles.lineChartContainer}>
-        <LineChart data={chartData.flat()} options={chartOptions} />
-      </div>
-    </div>
+      {config.interpretationSlot ? (
+        <div>
```

Comment:
nit: Could probably do without this wrapping div. Overall, LGTM. Curious to see an example of what would go into this slot.

Reply:
I did a wrapper because of the displaying errors I was having. Without it, it wouldn't render under the graph but on the side of it. 
Sorry for the late response @denniskigen .